pages/_app.js

```diff
@@ -1,28 +1,24 @@
 import Navigation from "@/components/Navigation";
 import GlobalStyle from "../styles";
 import { useEffect, useState } from "react";
-import plants from "@/assets/plants";
 import useLocalStorageState from "use-local-storage-state";
 import { useRouter } from "next/navigation";
```

Comment:
```suggestion
import { useRouter } from "next/router";
```
Verwendet hier bitte diesen Pfad (da ihr das sog. routing über den 'pages' folder macht. Der andere Pfad ist für den sog. 'app' folder optimiert.